project/api/models.py

```diff
@@ -68,8 +68,8 @@ class PoliticalRelation(models.Model):
     child = models.ForeignKey(
         TerritorialEntity, related_name="parents", on_delete=models.CASCADE
     )
-    start_date = models.DateField()
-    end_date = models.DateField()
+    start_date = models.DecimalField(decimal_places=1, max_digits=8)
+    end_date = models.DecimalField(decimal_places=1, max_digits=8)
```

Comment:
> In the latest commit we're storing each date as a double precision field with a max decimal place of 1, assuming we would follow jdcals method of storing JD for half day from calendar date.

Width decimal place of 1 it allows us to set time with 144 minutes precision.
It might not be precise enough if we want to store `time`, I would advice to go with `decimal_places=2` at least in this case. However I don't think we need to store `time` at all. It might be really confusing for user and it will possibly cause some blank areas.

If it is not about storing time then I am fine with using either integers (same as postgres) or double precision (we are not doing any math with them only gt/lt operations). 

I have checked some entities. ([api query](https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q488&format=json&languages=ru|en&origin=*))

timestamp by hour
[Hiroshima and Nagasaki atomic bombing](https://www.wikidata.org/wiki/Q488)
[Hiroshima atomic bombing](https://www.wikidata.org/wiki/Q703203)
[9.11](https://www.wikidata.org/wiki/Q10806)
[death of Kennedy](https://www.wikidata.org/wiki/Q9696)

Set only by date
[Franz Ferdinand's death](https://www.wikidata.org/wiki/Q43063)
[Chernobyl disaster](https://www.wikidata.org/wiki/Q486)
[renaming of Astana](https://www.wikidata.org/wiki/Q1520)

Reply:
Hmm, okay, I don't envision hourly step intervals in the global narrative for now so I guess we'll go IntField for all for now